src/Raven.Studio/typescript/models/resources/serverDashboard/storagePieChart.ts

```diff
@@ -129,15 +124,21 @@ class storagePieChart {
             arcs.select("path")
                 .transition()
                 .attrTween("d", arcTween)
-                .attr("class",d => this.colorClassScale(d.data.Database));
+                .attr("class", d => this.getDatabaseColorForPie(d.data.Database));
         } else {
             arcs.select("path")
                 .attr("d", d => arc(d as any))
-                .attr("class", d => this.colorClassScale(d.data.Database));
+                .attr("class", d => this.getDatabaseColorForPie(d.data.Database));
         }
+    }
+    
+    private getDatabaseColorForPie(dbName: string): string {
+        return dbName === '<System>' ? "color-11" : this.colorClassScale(dbName);        
```

Comment:
please use double quotes whereever possible (except html file in knockout bindings as it is inside ")